storage/command.c

```diff
@@ -0,0 +1,35 @@
+#include "command.h"
+
+int redis_del(redisContext* c, const char* const key) {
+  redisReply* reply = redisCommand(c, "DEL %s", key);
+  if (reply->integer) {
+    freeReplyObject(reply);
+    return 0;
+  } else {
```

Comment:
You don't have to put `else` statement since the condition is deterministic here. That is, we shall always write as following:
```c
if (reply->integer) {
    ...
    return 0;
}
/* do what you wish for condition !reply->integer */
```